docs/using-the-jdbc-driver/HostAvailabilityStrategy.md

```diff
@@ -0,0 +1,23 @@
+|------------------------------------------------|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Simple Host Availability Strategy              | `null` or blank      | This strategy is a simple pass through strategy. It does nothing when availability is set, and returns the raw host availability when fetched.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | 
```

Comment:
This is the first mention of raw host availability, can you elaborate on what that means?

Reply:
So raw host availability is just the host's availability without any extra logic or functionality. I will remove that wording and simply say that this strategy does not add any extra logic and functionality.

Reply:
Done!